Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs

```diff
@@ -202,16 +203,13 @@ async UniTaskVoid ShowTipWithFadeAsync(CancellationToken ct)
                 int prevTip = currentTip;
                 currentTip = index;
 
-                var backgroundTask = viewInstance.Background.DOBlendableColor(tips.Tips[index].BackgroundColor, 1f)
-                                                 .ToUniTask(cancellationToken: ct);
 
-                async UniTask HideAndShowAsync()
-                {
-                    await viewInstance.HideTipWithFadeAsync(prevTip, 0.5f, ct);
-                    await viewInstance.ShowTipWithFadeAsync(index, 0.5f, ct);
-                }
+                await viewInstance.HideTipWithFadeAsync(prevTip, 0.3f, ct);
 
-                await UniTask.WhenAll(backgroundTask, HideAndShowAsync());
+                await UniTask.WhenAll(
+                    viewInstance.ChangeBackgroundColor(viewInstance.Background.material, "_Color1", tips.Tips[index].BackgroundColor, 0.3f, ct),
```

Comment:
Feature envy smell. You should be able to do: `viewInstance.SetBackgroundColor(tips.Tips[index].BackgroundColor, 0.3f, ct)`

Reply:
I want to wait until the other tip is hidden to change the background color and display the tip at the same time

Reply:
Its ok, you can make `viewInstance.ChangeBackgroundColor` function an async task